single_stage_detector/ssd/run_and_time.sh

```diff
@@ -44,8 +44,8 @@ LR=${LR:-"2.5e-3"}
 
 echo "running benchmark"
 
-export DATASET_DIR="/data/coco2017"
-export TORCH_MODEL_ZOO="/data/torchvision"
+export DATASET_DIR=${DATASET_DIR:-"/data/coco2017"}
+export TORCH_MODEL_ZOO=${TORCH_MODEL_ZOO:-"/data/torchvision"}
```

Comment:
I believe the TORCH_MODEL_ZOO variable is unnecessary if you are pre-downloading the resnet34-333f7ec4.pth file and using the `--pretrained_backbone` script argument.  Also Pytorch changes the name of the envvar in almost every version, so you can't really depend on it (which is part of why we implemented the `--pretrained_backbone` script argument).